src/internet_identity/src/main.rs

```diff
@@ -414,10 +412,10 @@ fn save_persistent_state() {
 
 fn update_root_hash() {
     use ic_certified_map::{fork_hash, labeled_hash};
-    state::asset_hashes_and_sigs(|asset_hashes, sigs| {
+    state::assets_and_signatures(|assets, sigs| {
         let prefixed_root_hash = fork_hash(
-            // NB: Labels added in lexicographic order
-            &labeled_hash(LABEL_ASSETS, &asset_hashes.root_hash()),
+            &assets.root_hash(),
+            // NB: sigs have to be added last due to lexical ordering of labels
```

Comment:
nit: I'm not a native speaker, but I'd keep "lexicographic ordering" here.
(cf. https://wikidiff.com/lexical/lexicographic)